api/src/pcapi/routes/backoffice_v3/templates/admin/roles.html

```diff
@@ -1,7 +1,27 @@
 {% extends "layouts/admin.html" %}
 {% block roles_management %}
-  <div class="container">
+  <div class="container pc-role">
     <h2>Gestion des rôles</h2>
+    <div class="row">
+      <div class="col-12">
+        <form class="input-group my-3 px-1">
+          <span class="input-group-text bg-white"><i class="bi bi-search"></i></span>
+          <input type="text"
+                 class="form-control col-8 border-start-0"
+                 name="q"
+                 value=""
+                 data-search-role
+                 data-search-target=".pc-role"
+                 aria-describedby="search-icon"
+                 placeholder="Gérer les offres, ..." />
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Changer le placeholder

Reply:
Ajout du suffix `Ex :`